code/style.css

```diff
@@ -1,5 +1,172 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
 
 body {
-  background-color: peachpuff;
+  font-family: "Courier New", Courier, monospace;
+}
+
+header {
+  margin: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+footer {
+  margin: 40px 40px;
```

Comment:
If you want to have the same value for both horizontal margins as well as vertical, all you need is `margin: 40px;` 😊👌